docs/5.x/cluster.md

```diff
@@ -2124,6 +2026,89 @@ $ gravity resource rm runtimeenvironment
     of runtime containers on each cluster node. Take this into account and plan each update accordingly.
 
 
+### Configuring Cluster
+
+It is possible to perform last-minute configuration before the installation or update some aspects of the cluster
```

Comment:
I would consider re-wording this, not as last-minute configuration, but maybe as customizing the cluster per environment?